code/datums/diseases/advance/symptoms/clockwork.dm

```diff
@@ -0,0 +1,436 @@
+/datum/symptom/robotic_adaptation
+	name = "Biometallic Replication"
+	desc = "The virus can manipulate metal and silicate compounds, becoming able to infect robotic beings. The virus also provides a suitable substrate for nanites in otherwise inhospitable hosts"
+	illness = "Robotic evolution"
+	stealth = 0
+	resistance = 1
+	stage_speed = 4 //while the reference material has low speed, this virus will take a good while to completely convert someone
+	transmittable = -1
+	level = 8
+	severity = 0
+	symptom_delay_min = 10
+	symptom_delay_max = 30
+	//var/prefixes = list("Ratvarian ", "Keter ", "Clockwork ", "Robo")
+	var/bodies = list("Robot")
+	//var/suffixes = list("-217")
+	var/replaceorgans = FALSE
+	var/replacebody = FALSE
+	var/robustbits = FALSE
+	threshold_descs = list(
+		"Stage Speed 4" = "The virus will replace the host's organic organs with mundane, biometallic versions.",
+		"Resistance 4" = "The virus will eventually convert the host's entire body to biometallic materials, and maintain its cellular integrity.",
+		"Stage Speed 10" = "Biometallic mass created by the virus will be superior to typical organic mass."
+	)
+/datum/symptom/robotic_adaptation/OnAdd(datum/disease/advance/A)
+	A.infectable_biotypes |= MOB_ROBOTIC
+
+///datum/symptom/robotic_adaptation/severityset(datum/disease/advance/A)
+//	. = ..()
+//	if(A.totalStageSpeed() >= 4) //at base level, robotic organs are purely a liability
+//		severity += 1
+//		if(A.totalStageSpeed() >= 10)//but at this threshold, it all becomes worthwhile, though getting augged is a better choice
+//			//severity -= 3//net benefits: 2 damage reduction, flight if you have wings, filter out low amounts of gas, durable ears, flash protection, a liver half as good as an upgraded cyberliver, and flight if you are a winged species
+//	if(A.totalResistance() >= 4)//at base level, robotic bodyparts have very few bonuses, mostly being a liability in the case of EMPS
+//		severity += 1 //at this stage, even one EMP will hurt, a lot.
+
+
+/datum/symptom/robotic_adaptation/Start(datum/disease/advance/A)
+	. = ..()
+	if(A.totalStageSpeed() >= 4)
+		replaceorgans = TRUE
+	if(A.totalResistance() >= 4)
+		replacebody = TRUE
+	if(A.totalStageSpeed() >= 10)
+		robustbits = TRUE //note that having this symptom means most healing symptoms won't work on you
+
+
+/datum/symptom/robotic_adaptation/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/human/H = A.affected_mob
```

Comment:
single letter variable bad

Reply:
do you know how many single letter variables virology system got, welp time to rewrite them all over the place.